img/ico-discord.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="71" height="55" viewBox="0 0 71 55" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This icon was added when someone updated the chat link.  I don't see references to the other icons from the old site, but the files are still here, so I kept it on the merge.